web/src/engine/osk/src/input/gestures/browser/flick.ts

```diff
@@ -118,7 +118,7 @@ export default class Flick implements GestureHandler {
     let source: GestureSource<KeyElement> = baseSource;
 
     sequence.on('complete', () => {
-      previewHost.cancel()
+      previewHost?.cancel()
```

Comment:
Threw it in since I saw a null-ref error once while testing.

Reply:
Are there any reports on Sentry we can link?

Reply:
My local build that was in place when the error occurred does not appear to have sent a Sentry log, unfortunately.

Reply:
And no other reports from other people... a little surprising!

Reply:
Aha!  We _do_ have one now:

https://keyman.sentry.io/share/issue/9d0e6b45e99a4afab5b4425e603c35e4/ (it was under Developer)